modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1740,78 +1820,106 @@ public void testUpdateSystemObjectDefinition() throws Exception {
 
 		// Modifiable system object definition must be published to be actived
 
-		_assertFailure(
+		AssertUtils.assertFailure(
 			ObjectDefinitionActiveException.class,
 			"Object definitions must be published before being activated",
-			objectDefinition,
-			objectDefinitionId ->
-				_objectDefinitionLocalService.updateCustomObjectDefinition(
-					null, objectDefinitionId, 0, 0, 0, 0, false, true, false,
-					true, false, false,
-					LocalizedMapUtil.getLocalizedMap("Charlie"), "Charlie",
-					null, null, false,
-					LocalizedMapUtil.getLocalizedMap("Charlies"),
-					ObjectDefinitionConstants.SCOPE_SITE));
+			() -> _objectDefinitionLocalService.updateCustomObjectDefinition(
+				null, objectDefinition1.getObjectDefinitionId(), 0, 0, 0, 0,
+				false, true, false, true, false, false,
+				LocalizedMapUtil.getLocalizedMap("Charlie"), "Charlie", null,
+				null, false, LocalizedMapUtil.getLocalizedMap("Charlies"),
+				ObjectDefinitionConstants.SCOPE_SITE));
 
 		// Label is null
 
-		_assertFailure(
+		AssertUtils.assertFailure(
 			ObjectDefinitionLabelException.class,
 			"Label is null for locale " + LocaleUtil.US.getDisplayName(),
-			objectDefinition,
-			objectDefinitionId ->
-				_objectDefinitionLocalService.updateCustomObjectDefinition(
-					null, objectDefinitionId, 0, 0, 0, 0, false, false, false,
-					true, false, false, null, "Charlie", null, null, false,
-					LocalizedMapUtil.getLocalizedMap("Charlie"),
-					ObjectDefinitionConstants.SCOPE_SITE));
+			() -> _objectDefinitionLocalService.updateCustomObjectDefinition(
+				null, objectDefinition1.getObjectDefinitionId(), 0, 0, 0, 0,
+				false, false, false, true, false, false, null, "Charlie", null,
+				null, false, LocalizedMapUtil.getLocalizedMap("Charlie"),
+				ObjectDefinitionConstants.SCOPE_SITE));
+
+		// Object folder does not exist
+
+		long objectFolderId = RandomTestUtil.randomLong();
+
+		AssertUtils.assertFailure(
```

Comment:
Can we move this case to the top after the objectDefinition1 initialization?

Reply:
Agreed